### DISCUSSION TITLE
Storage Estimation for Elasticsearch with Security Onion

### DISCUSSION THREAD

**Discussion by veraxcyber** [Security-Onion-Solutions/securityonion#3914] [Unsupported Versions]
Hello, I am looking to start a managed security provider for very small businesses with roughly one server and 10 hosts. 
I will be looking to build this out to more customers over time. 

What is the best way to determine an estimated amount of storage required to confirm the cost of hosting in Elastic Cloud?

I will be using Suricata as the NIDS, Wazuh as the HIDS, Windows Logs, Antivirus Logs, Sysmon Logs and flow logs. 

 I understand this can change but if there was a way to determine an estimate that would be greatly appreciated. 

Many Thanks!

**Comment by Git-Me-Some-Hub** [comment]
Do you have retention requirements? 
What type of NIDS logs are you getting, only local LAN traffic or are you connecting a SPAN Port? For me running at my house the SPAN ports was a pacman of storage space. I do not have good numbers for you, but I filled a 2 TB drive in around 45-60 days.

**Comment by veraxcyber** [comment]
Hello,

Thanks for the reply, yeah I have a SPAN port with mirroring the traffic on the network.

How many devices is that collecting traffic for?

Many Thanks!
